src/arb/ArbTypes.jl

```diff
@@ -867,17 +867,7 @@ base_ring(a::AcbPolyRing) = a.base_ring
 #
 ################################################################################
 
-struct RealMatSpace <: MatSpace{RealFieldElem}
-  nrows::Int
-  ncols::Int
-
-  function RealMatSpace(R::RealField, r::Int, c::Int)
-    (r < 0 || c < 0) && throw(_err_dim_negative)
```

Comment:
Ah, and we can now delete `_err_dim_negative` again as there are no references to it left in Nemo.